pages/blog/fetch-for-servers.mdx

```diff
@@ -0,0 +1,182 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'JavaScript runs everywhere, so should your servers - here is how',
```

Comment:
```suggestion
  title: 'JavaScript runs everywhere, so should your HTTP server - here is how',
```

I think this is more specific

Reply:
Hmm we need a new image then @Urigo @charlypoly What do you think?

Reply:
I guess we can, let me know if you need a new set of images

Reply:
I prefer the existing title.....